### DISCUSSION TITLE
Can Istio service mesh support https->https?

### DISCUSSION THREAD

**Discussion by hrishikeshgjoshi** [istio/istio#46641] [Q&A]
Does Istio support TLS communication between envoy proxy to service running in pod for external-inbound traffic. As described in envoy documentation https://www.envoyproxy.io/docs/envoy/latest/start/sandboxes/tls
Ingress to istio envoy proxy is doing mTLS and TLS is terminated at proxy then can it start new TLS from proxy to service, where service is listening on HTTPS. Thus, proxy to 'Service A' communication also starts over TLS. In other words, can the service be hosted with https.
I am referring this image from istio documentation where we want to have local inbound over TLS only
https://istio.io/latest/docs/ops/configuration/traffic-management/tls-configuration/sidecar-connections.svg

**Comment by zirain** [comment]
is [this](https://istio.io/latest/docs/reference/config/networking/virtual-service/#TLSRoute) what you're looking for?
